### ISSUE TITLE
Information in "No" list item result is not clearly visible after changing the orientation from landscape to portrait mode.

### ISSUE THREAD

**Issue by tripathiakhil429** [microsoft/vscode#143277]
**GitHub Tags:**#A11yMAS;WCAG1.3.4;A11ySev2;#A11yTCS;#VisualStudioCode-Win32-Apr2021;#Visual Studio Code Client;#A11yWCAG2.1;#DesktopApp;#Win10
### Environment Details:
Visual studio code  
kubernetes
Windows Version: Windows11
### Repro Steps:

1. Setup The Kubernetes extension.
2. Open visual studio code and load "Bikes" project.
3. Tab to "View" menu item in top navigation bar, press enter to expand and use down arrow key to Select "Command Pallet" option.
4. Search and select "Local Process with Kubernetes: Configure Local Process with Kubernetes”
5. Connect to Kubernetes panel will open.
6. Tab to bike options and press enter
7. Input “3000” as your local port and press enter.
8. Tab to Launch via NPM option and press enter to select it.
9. Change the orientation from landscape and portrait mode
10. Observe that the content in no result is not visible after changing the orientation from landscape to portrait mode.

### Actual:
Information in "No" list item result is not clearly visible after changing the orientation from landscape to portrait mode.

### Expected:
Information in "No" list item result should be completely visible after changing the orientation from landscape to portrait mode..
### User Impact:
Low vision users will not be able to view the Information of No list item

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!
